tests/vmi_lifecycle_test.go

```diff
@@ -1613,13 +1613,18 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Signaled graceful shutdown for %s", vmi.GetObjectMeta().GetName())), "Should log graceful shutdown")
 
 				// Verify VirtualMachineInstance is killed after grace period expires
-				By("Checking that the VirtualMachineInstance does not exist after grace period")
-				Eventually(func() string {
-					data, err := logsQuery.DoRaw(context.Background())
-					Expect(err).ToNot(HaveOccurred(), "Should get logs")
-					return string(data)
-				}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Grace period expired, killing deleted VirtualMachineInstance %s", vmi.GetObjectMeta().GetName())), "Should log graceful kill")
-			})
+				if shouldBeKilled {
+					By("Checking that the VirtualMachineInstance does not exist after grace period")
+					Eventually(func() string {
+						data, err := logsQuery.DoRaw(context.Background())
+						Expect(err).ToNot(HaveOccurred(), "Should get logs")
+						return string(data)
+					}, 30, 0.5).Should(ContainSubstring(fmt.Sprintf("Grace period expired, killing deleted VirtualMachineInstance %s", vmi.GetObjectMeta().GetName())), "Should log graceful kill")
+				}
+			},
+				Entry("[test_id:1655] gracefully shutdown", decorators.WgS390x, libvmifact.NewFedora(), console.LoginToFedora, false),
```

Comment:
@nekkunti Is there an option for the guest to ignore shutdown request on the s390x arch? This test is specifically testing that the graceful shutdown is ignored by the guest (Alpine doesn't have ACPI daemon).  

The `[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should result in vmi status succeeded` test actually test what you test with this modification. I acknowledge the tests are not that clear, I am working on this.

Reply:
/hold 
This is wrong

Reply:
>>>Is there an option for the guest to ignore shutdown request on the s390x arch? 
as per my knowledge, we dont have  , but mostly testing  gracefull shudown for s390x rathan than failure  case)
i have spint into 2 test cases 
1.  by using fedora which has qemu agent/acpid so that gracefull shurdown done properly. (label for s390x also , so that it will run for s390x)
2.  same as old test with alpine( when acpi daemon not there , it wil iniate garacefull shutdown but it fail/ignore then we are initiating force shudown by kill signal)

i didn't see any issue 

can ypou explain me why you think this is wrong .

Reply:
You basically convert this test to test something totally different. It basically turns into `[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should result in vmi status succeeded` test.

I will ask again "Is there an option for the guest to ignore shutdown request on the s390x arch?".  For example how does guestless VM behave? Does the firmware waits or it shutdown?

 Let's just skip this test for s390x if there is no such option.

Reply:
Thanks @xpivarc  ,   No .. i did't find anything to ignore for s390x arch .   Guestless VM   is not working in s390x case  , it seems some issue with qemu .we are checking ( [link](https://kubernetes.slack.com/archives/C0163DT0R8X/p1726468656344359).) 

I am ok with skipping for s390x  for now.

Reply:
Maybe you can tag `[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should result in vmi status succeeded` if it is not already...

Reply:
> Maybe you can tag `[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should result in vmi status succeeded` if it is not already...

what do mean tagging ? , I did't get copletely , Please could you explain bit more or give me pseudo code

Reply:
Tagging/labeling. Here you put label `decorators.WgS390x` so it will be run within the s390x. My suggestion is to make sure the test I mention is included and that should verify graceful shutdown is working on s390x.

Reply:
oh .. ok , got it .  Thanks @xpivarc  ,  this test name misled me

Reply:
Done